src/github.rs

```diff
@@ -749,6 +749,21 @@ impl Issue {
         Ok(Some(String::from(String::from_utf8_lossy(&diff))))
     }
 
+    /// Returns the commits from this pull request (no commits are returned if this `Issue` is not
+    /// a pull request).
+    pub async fn commits(&self, client: &GithubClient) -> anyhow::Result<Vec<Commit>> {
+        if !self.is_pr() {
+            return Ok(vec![]);
+        }
+
+        let req = client.get(&format!(
+            "{}/pulls/{}/commits",
```

Comment:
Note that this will only return at most 30 by default, I'd guess -- I think we should paginate here (shouldn't be too hard, can steal code from elsewhere in the file most likely). It's less important for this specific use case but could bite us later.

(Probably pages of 100 make sense).

Reply:
Added support for pagination.